db/seeds.rb

```diff
@@ -64,13 +64,13 @@
 Planhabit.create(plan_id: plan3.id, habit_id: habit20.id)
 Planhabit.create(plan_id: plan3.id, habit_id: habit21.id)
 
-plan4 = Plan.create(plan_name: "Launch Prep-Beastmode", author: "Paul M. Thiebauth", author_info: "Web Developer - Boston, MA")
+plan4 = Plan.create(plan_name: 'Launch Prep-Beastmode', author: 'Paul M. Thiebauth', author_info: 'Web Developer - Boston, MA')
```

Comment:
Line is too long. [127/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.